skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/routes/Land.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.pensjon.brev.skribenten.routes
+
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import java.util.*
+
+internal val norskBokmål: Locale = Locale("NB", "NO")
+internal val landKoder: Set<String> = Locale.getISOCountries(Locale.IsoCountryCode.PART1_ALPHA2)
+internal val land = landKoder.map {
+    LandJson(it, Locale("", it).getDisplayCountry(norskBokmål))
+}
+
+internal data class LandJson(
+    val kode: String,
+    val navn: String
+)
+
+//TODO - test
+fun Route.landRoute() {
+    get("/land") {
```

Comment:
Litt usikker på om `/land` er et godt valg. Men samtidig så klarer jeg ikke helt å komme med et bedre forslag. Grunnen til at jeg ikke liker `/land` som rotsti er at det er en form for metadata, kodeverk e.l., altså ikke en ressurs i Skribenten og er mer for validering  enn noe annet.

Reply:
Jeg klarte heller ikke å komme på et godt alternativ, så jeg bare 'land'et på det enkleste for å få det gående. vi kan sikkert muligens prefikse den med noe?